libs/dscache/odc/dscache/apps/dstiler.py

```diff
@@ -14,8 +14,12 @@
 @click.option('--native', is_flag=True, help='Use Landsat Path/Row as grouping')
 @click.option('--native-albers', is_flag=True, help='When datasets are in Albers grid already')
 @click.option('--web', type=int, help='Use web map tiling regime at supplied zoom level')
+@click.option('--crs', type=str, help="Custom gridspec: crs")
+@click.option('--resolution', type=str, help="Custom gridspec: resolution_y,resolution_x")
+@click.option('--shape', type=str, help="Custom gridspec: shape_y,shape_x")
```

Comment:
Can we change help string to just say `Y,X` rather than `{resolution|shape}_{y,x}`, I feel like that read clearer repeating same word three times on one line of text.

"Custom gridspec: resolution in CRS units per pixel in y,x order"
"Custom gridspec: shape of tile in pixels ny,nx"